addons/web_tour/static/src/tour_service/tour_service.js

```diff
@@ -63,7 +63,7 @@ export const tourService = {
         const tours = {};
         const tourRegistry = registry.category("web_tour.tours");
         function register(name, tour) {
-            tours[name] = {
+            tours[tour.saveAs || name] = {
```

Comment:
```suggestion
            name = tour.saveAs || name
            tours[name] = {
            name,
            
```
To avoid another error